tsdb/compact_test.go

```diff
@@ -1632,7 +1633,7 @@ func TestSparseHistogramSpaceSavings(t *testing.T) {
 				}()
 
 				wg.Add(1)
-				go func() {
+				go func(c testcase) {
```

Comment:
Is this necessary now that Prometheus requires Go v1.22, which [prevents](https://tip.golang.org/doc/go1.22) the loop variable capture bug?

Reply:
Maybe it's not necessary; it's still clearer.